.gitmodules

```diff
@@ -1,6 +1,9 @@
 [submodule "sim/mbedtls"]
 	path = ext/mbedtls
 	url = https://github.com/ARMmbed/mbedtls
+[submodule "sim/tinycrypt"]
+	path = ext/tinycrypt
```

Comment:
Should it be `sim/tinycrypt`?

Reply:
https://github.com/JuulLabs-OSS/mcuboot/pull/785#issuecomment-675348710